nomenclature/region_mapping_models.py

```diff
@@ -134,6 +137,12 @@ def all_regions(self) -> List[str]:
         cr_list = [x.name for x in self.common_regions or []]
         return nr_list + cr_list
 
+    @property
+    def renaming_dict(self) -> Dict[str, str]:
+        # NOTE: right now this implementation would imply that all not defined model
```

Comment:
I would put that explicitly into a documentation page (for now as part of the main Readme of the repo)...

Reply:
Added some explanation to the README